cleverhans/attacks.py

```diff
@@ -241,16 +250,23 @@ def parse_params(self, eps=0.3, ord=np.inf, y=None, clip_min=None,
                   labels to avoid the "label leaking" effect (explained in this
                   paper: https://arxiv.org/abs/1611.01236). Default is None.
                   Labels should be one-hot-encoded.
+        :param y_target: (optional) A tensor with the labels to target. Do not
+                         set y_target if y is also set. Labels should be
+                         one-hot-encoded.
```

Comment:
This wouldn't be that easy. If people use generate, then y_target is a tensor, and so we'd have to put an assert in the middle of the TF graph.